grommet/grommet
# Fix range selector step issue

#### What does this PR do?
Fixes issue with step on range selector with mouse interaction.

#### Where should the reviewer start?
src/js/components/RangeSelector/RangeSelector.js

#### What testing has been done on this PR?
The issue noted in Issue #3360 was recreated in a local project. Then after the fix, the changes were tested in a storybook example and the example provided in order to confirm successful changes.

#### How should this be manually tested?
Create a RangeSelector with a step value and ensure that as the mouse moves either end of the range selector that the value is adhering to the defined step.

#### Any background context you want to provide?
The RangeSelector was creating a glitchy effect when a step was provided. For example, if the step was 3, the RangeSelector output value would jump from 0 -> 2 -> 4 -> 3 -> 5 -> 7 -> 6 and so on. This does not achieve the intended goal of only allowing the RangeSelector to reach values in increments of 3.

#### What are the relevant issues?
Issue #3360 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, this is backwards compatible.
